js/common/tests/shell/shell-view-arangosearch-link-noncluster.js

```diff
@@ -76,10 +80,55 @@ function IResearchLinkSuite () {
       var meta = { links: { 'testCollection' : { includeAllFields: true } } };
       var view = db._createView("badView", "arangosearch", meta);
       var links = view.properties().links;
-      assertEqual(links['testCollection'], undefined);
+      assertNotEqual(links["testCollection"], undefined);
       view.drop();
     },
 
+    testHandlingCreateWithMultipleLinks : function () {
+      var meta = { links: { 'testCollection' : { includeAllFields: true },
+        'testCollection2' : { includeAllFields: true } } };
+      var view = db._createView("badView", "arangosearch", meta);
+      var links = view.properties().links;
+      assertNotEqual(links["testCollection"], undefined);
+      assertNotEqual(links["testCollection2"], undefined);
+      view.drop();
+    },
+
+    testHandlingCreateWithBadLinks : function () {
+      var meta = { links: { 'nonExistingCollection' : { includeAllFields: true } } };
+
+      try {
+        var view = db._createView("badView", "arangosearch", meta);
+      } catch(err) {
+        assertEqual(ERRORS.ERROR_BAD_PARAMETER.code, err.errorNum);
+      }
+
+      try {
+        var view = db._view('badView');
+      } catch (err) {
+        assertEqual(ERRORS.ERROR_VIEW_NOT_FOUND.code, err.errorNum);
```

Comment:
if an exception is expected then the non-exceptional branch should fail the test